common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -1073,7 +1064,7 @@ def has_course(self, course_key, ignore_case=False, **kwargs):
                     course_query[key] = re.compile(r"(?i)^{}$".format(course_query[key]))
         else:
             course_query = {'_id': location.to_deprecated_son()}
-        course = self.collection.find_one(course_query, fields={'_id': True})
+        course = self.collection.find_one(course_query)
```

Comment:
I made this change after my reading of the docs told me that no `fields` would return the same as this. A re-reading of the docs tells me that I'm incorrect - this will return only the `_id` and nothing else.

I'm going to change this back - but make it compatible with PyMongo 3.0.